### ISSUE TITLE
UI Component - Record Audio

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#13515]
This issue is to track the creation of the component: **Record Audio**

You can access the link at https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=3289%3A63901.

Please have a chat with the design team before starting with the component, and don't forget to add sizing to the issue

**Comment by briansztamfater** [comment]
I opened a new issue to handle the audio recording flow in two parts. On this one I implemented recording button with its animations and interactions in #14170, and will use #14558 to finish the full recording component / flow using the brand new recording button.
